python/ray/autoscaler/log_timer.py

```diff
@@ -5,13 +5,19 @@
 
 
 class LogTimer:
-    def __init__(self, message):
+    def __init__(self, message, show_status=True):
```

Comment:
if there's other usage of LogTimer, you'll probably want to default this to False

Reply:
Sure!